Frontend/src/views/Login.vue

```diff
@@ -31,8 +31,10 @@
                     password: this.$data.password
                })
                .then(resp => {
-                   localStorage.email = this.email;
-                   localStorage.token = resp.data; 
+                   let respData = resp.data
+                   localStorage.setItem('email', this.$data.email)
+                   localStorage.setItem('token', respData)
+                   this.$bus.$emit('logged', 'User logged')
```

Comment:
You should not need an eventbus for this feature.

Reply:
I did do it at first without using an event bus but you had to refresh the page in order to see the change.

Reply:
It's possible to do this without event bus. Just share the isLoggedIn variable in a shared service.

Reply:
so like using Vuex? I believe Vuex works as a shared service.

Reply:
No need for vuex either. Localstorage should do everything you need concerning state.